services/core-api/app/api/mines/project_summary/response_models.py

```diff
@@ -17,7 +17,9 @@
         'status_code': fields.String,
         'project_summary_lead_party_guid': fields.String,
         'project_summary_lead_name': fields.String,
-        'documents': fields.List(fields.Nested(PROJECT_SUMMARY_DOCUMENT_MODEL))
+        'documents': fields.List(fields.Nested(PROJECT_SUMMARY_DOCUMENT_MODEL)),
+        'update_user': fields.String,
+        'update_timestamp': fields.DateTime
     })
```

Comment:
We don't usually return the audit columns to the FE, we've created hybrid properties or created duplicate columns in the past that we expose to the UI. Our current pattern is not good though and given this is an expanding need of the users I think we should re-assess how we are doing it. 

@JustinMacaulay thoughts?